src/private_cuda.h

```diff
@@ -107,7 +107,8 @@ GPUARRAY_LOCAL int cuda_record(gpudata *, int);
 /* private flags are in the upper 16 bits */
 #define CUDA_WAIT_READ  0x10000
 #define CUDA_WAIT_WRITE 0x20000
-#define CUDA_WAIT_MASK  0x30000
+#define CUDA_WAIT_FORCE 0x40000
```

Comment:
Then it might need that.  But we don't have any such ops for now and I don't think we will have any in the future.  I don't plan for "what ifs".